src/page/Login.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from "prop-types";
+
+import Input from "../component/form/Input";
+
+import './login.scss';
+
+const Login = ({handleSubmit, onChange, form}) => {
+
+    return (<>
+        <div className="login__background"/>
+        <section className="login__card__frame">
+            <article className="login__card">
+                <div className="login__card__body">
```

Comment:
Is this div necessary? I think we can remove this if there's no specific reason to have it.